app/models/order.py

```diff
@@ -3,22 +3,21 @@
 from collections import defaultdict
 from datetime import datetime
 import secrets
-import string
 
 from hlds.definitions import location_definitions
 from utils import first
 
 from .database import db
 from .user import User
 
-BASE31_ALPHABET = '23456789abcdefghjkmnpqrstuvwxyz'
 
 def generate_slug():
-    secret = ''.join(secrets.choice(BASE31_ALPHABET) for i in range(8))
+    secret = secrets.token_urlsafe(24)
```

Comment:
Fyi token_urlsafe generates 24 bytes and formats it as base64, which results in a character length of 32, so the field is 32 chars.